### DISCUSSION TITLE
Format column/text fixed with 'Text'

### DISCUSSION THREAD

**Discussion by Thiery176** [abap2xlsx/abap2xlsx#1241] [Q&A]
Hello all,

i am new to ABAP2XLSX. I was playing around with it but i did not manage to do one thing. I would like to have as a columns format (or cell format if not possible in another way) not the 'Standard' but the 'Text':

![image](https://github.com/abap2xlsx/abap2xlsx/assets/173345488/f9b5ac5b-9475-481b-9ab4-eafb6c28cbfe)

So, that people that maintain the excel after the download do not have to fight with loosing leading zeros:

![image](https://github.com/abap2xlsx/abap2xlsx/assets/173345488/d5b34b41-9ad1-4dbc-84de-9b0bebdbc971)

Can you give me a hint if it is possible and if so: how?

Many thanks for any suggestions!

Frank T.

**Comment by sandraros** [comment]
Hi, it's all about "format". You have these demo programs for instance:
![image](https://github.com/abap2xlsx/abap2xlsx/assets/34005250/0b999c56-4bcb-4585-881c-b1e4a6a1c858)

**Reply by Thiery176** [reply]
Super...I was able to adopt it and it already works in my use-case. I was aware about the Demos and I had also a look at many, but somehow I was not able to find/understand it.  

Many thanks for your prompt help!

Frank T.
